src/packages/solid/components/Row/Row.tsx

```diff
@@ -0,0 +1,66 @@
+import { Component, createEffect, on } from 'solid-js';
+import { View, activeElement, type IntrinsicNodeProps } from '@lightningjs/solid';
+import { Row as SolidRow } from '@lightningjs/solid-primitives';
+import styles from './Row.styles';
+import { getScrollValue } from 'utils';
+import theme from 'theme';
+
+export interface RowProps extends IntrinsicNodeProps {
+  /**
+   * components to be listed in the Row
+   */
+  children: object;
+  /** Item index at which scrolling begins */
+  scrollIndex?: number;
+
+  scrollStyle?: 'alwaysScroll' | 'neverScroll' | 'lazyScroll' | undefined;
```

Comment:
we should probably call this something else, we use the term `style` to reference a pretty specific thing. maybe `ScrollType` or something else?